### ISSUE TITLE
Native crash on macos

### ISSUE THREAD

**Issue by daniorio** [GTNewHorizons/lwjgl3ify#116]
[message(14).txt](https://github.com/GTNewHorizons/lwjgl3ify/files/14177480/message.14.txt)
[hs_err_pid6423.log](https://github.com/GTNewHorizons/lwjgl3ify/files/14177482/hs_err_pid6423.log)

**Comment by daniorio** [comment]
> Were you resizing the window or otherwise changing the viewport? See [GTNewHorizons/GT-New-Horizons-Modpack#14588](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/14588)

No, the crash happens before the window even appears on startup

**Comment by daniorio** [comment]
And this crash happens not sometimes, as described here (https://github.com/glfw/glfw/issues/1997), but at any launch on the mac

**Comment by eigenraven** [comment]
I see that you ran minecraft through some launcher that intercepts and somewhat reconfigures the LWJGL init process, can you replicate the same crash in Prism launcher or MultiMC?
